schemas/preferences.sublime-settings.json

```diff
@@ -217,7 +217,7 @@
           "default": "auto"
         },
         "rulers": {
-          "markdownDescription": "Columns in which to display vertical rulers.\n\nAs of ST 4133 column and style (see `ruler_style`) can be paired.\n\ne.g.: `\"rulers\": [[80, \"solid\"], [100, \"dotted\"]]`",
+          "markdownDescription": "Columns in which to display vertical rulers. Each entry may also be a list of the column, the ruler style and optionally the ruler width, for example: `[[80, \"solid\", 2], 78, [90, \"dotted\"]]`",
```

Comment:
Not that if we want to separate paragraphs with `\n\n` then that's fine. I just meant hard wrapping at random point in the sentence.